### ISSUE TITLE
Initiate GUI from an icon instead of from command line

### ISSUE THREAD

**Issue by djfkahn** [djfkahn/MemberHubDirectoryTools#110]

**Comment by djfkahn** [comment]
Used the `pyshortcuts` library to create the desktop shortcut in the "shortcut_maker.py".  I used the Husky thumbnail for an icon, and labeled the short cut as "Husky MH Tools"

Here is the script content:
```
from pyshortcuts import make_shortcut

make_shortcut(script   = './gui_executive.py',
              name     = 'Husky MH Tools',
              icon     = './MH_Tools.icns',
              desktop  = True,
              terminal = False)
```

**Comment by djfkahn** [comment]
addressed in commit d16d421
